pkg/services/ngalert/remote/alertmanager_test.go

```diff
@@ -353,8 +358,8 @@ func TestIntegrationRemoteAlertmanagerApplyConfigOnlyUploadsOnce(t *testing.T) {
 	{
 		require.NoError(t, store.Set(ctx, cfg.OrgID, "alertmanager", "silences", base64.StdEncoding.EncodeToString([]byte("abc123"))))
 		require.NoError(t, store.Set(ctx, cfg.OrgID, "alertmanager", "notifications", base64.StdEncoding.EncodeToString([]byte("abc123"))))
-		fakeConfig.ID = 30000000000000000
```

Comment:
We're not using the ID anymore when sending a configuration, so changing the ID field won't change the payload we send. This fix is unrelated to this PR, but not worth making a separate PR.